### PR TITLE
#939 fixed

### PR DESCRIPTION
**Pull request by dilpreetsio**
Improved "Show starred" and "Search" field on mobile screen

### CODE REVIEW

**Comment by shubham-padia** [comment]
@geekyd Thanks. Please clear the hound-ci issues and provide us with a live link or relevant screenshots in the PR itself to make the review process faster.

**Comment by geekyd** [comment]
@shubham-padia I have resolved the hound-ci issues.

Here are the screenshots of the new UI.

screen shot 2017-01-05 at 11 29 25 am
screen shot 2017-01-05 at 11 29 30 am
screen shot 2017-01-05 at 11 29 37 am

File: src/backend/templates/tracks.hbs [hunk 1]
```diff
@@ -96,7 +96,7 @@
                               </i>
                             </a>
                           </div>
-                            
+
```
Comment (shubham-padia) [inline src/backend/templates/tracks.hbs]: @geekyd please remove this extra line :)

Comment (geekyd) [inline src/backend/templates/tracks.hbs]: I have removed extra lines in the latest commit. :)

Comment (shubham-padia) [inline src/backend/templates/tracks.hbs]: @geekyd thanks

File: src/backend/templates/tracks.hbs [hunk 2]
```diff
@@ -334,7 +333,7 @@
                             $(this).find('.bookmark').each(function() {
                                 $(this).css("color", "black");
                             });
-                        }
+ show-button-filter-filter            }
```
Comment (aayusharora) [inline src/backend/templates/tracks.hbs]: @geekyd What is `show-button-filter filter` here ? Please remove this.

Comment (aayusharora) [inline src/backend/templates/tracks.hbs]: Please solve line 336 mentioned  issue :)

File: src/backend/templates/rooms.hbs [hunk 3]
```diff
@@ -57,7 +57,7 @@
         {{#timeList}}
           <span class="date-pill"><a href="./rooms.html#{{slug}}"> {{date}} </a></span>
         {{/timeList}}
-        <span style = "display:inline-block; padding-top:6px;"> 
+        <span class="show-button-filter" >
```
Comment (aayusharora) [inline src/backend/templates/rooms.hbs]: Good work :)

Comment (geekyd) [inline src/backend/templates/rooms.hbs]: @aayusharora sorry about the redundant line , I may have pasted it by mistake . I will be more careful from now on.

Thank you! :)

**Comment by aayusharora** [comment]
@geekyd Please squash your commits :)

File: src/www/js/form.js [hunk 4]
```diff
@@ -15,13 +15,13 @@ $(document).ready(function () {
...
     var size = (event.file.size/(1024*1024)).toString().substring(0,3)
-    $('#upload-filesize').html( size + "M") 
+    $('#upload-filesize').html( size + "M")
   });
 
   $('#cancelUpload').click(function(e){
     //TODO cancel soket ongoing uploading of file
     e.preventDefault();
-    $('#siofu_input').val('').show()
+    $('#siofu_input').val('').show();
```
Comment (geekyd) [inline src/www/js/form.js]: @aayusharora Need your help unable to recognise the issue.

**Comment by Princu7** [comment]
@geekyd Great work. @mariobehling @aayusharora Merging this.

**Pull request merged**